src/components/Home.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios"
+import MediaCard from './MediaCard'
+
+export default function Home(props) {
+
+    const [pictureOfTheDay, setPicture] = useState({})
+
+    useEffect(() => {
+        axios.get('https://api.nasa.gov/planetary/apod?api_key=QK4MKQhWmFOjd13fkz22YyQX4bdE5Praodt56wm9')
```

Comment:
don't upload api_key to github. remove it from github (and github history), it should be added to .env file